dateparser/search/search.py

```diff
@@ -188,24 +195,32 @@ def detect_language(self, text, languages):
         else:
             self.language_detector = FullTextLanguageDetector(list(self.available_language_map.values()))
 
-        return self.language_detector._best_language(text)
+        detected_language = self.language_detector._best_language(text) or settings.DEFAULT_LANGUAGES[0]
```

Comment:
this is the place where I think ``DEFAULT_LANGUAGES`` is used even without custom language detection?

Reply:
I too believe the same

Reply:
I will make the required changes